launcher/src/components/UI/NodeBg.vue

```diff
@@ -1,5 +1,4 @@
 <template>
-
   <div id="parent">
```

Comment:
just using the parent name will make change to all elements in all componenmts, please make a more clear name maybe sidebar-parent